app/models/property.rb

```diff
@@ -138,6 +139,14 @@ class << self
 
   # public instance methods ...................................................
 
+  def host
+    URI.parse(url).host
+  end
+
+  def restrict_to_sponsor_campaigns?
+    host == "github.com"
```

Comment:
we probably do not want this. The GitHub Readme is used on an additional 180+ sites. We want it to work on all. We also will allow the sponsor placement on GitLab, etc.

Reply:
The sponsor will show wherever the readme is hosted. 

This logic is simply checking the canonical property domain. For example, consider a project who's canonical repo is at GitHub, but is also mirrored on Gitlab... and it's an NPM package. The property/project would be listed in CodeFund with the canonical GitHub URL, but the readme (including sponsor) will be rendered on GitHub, Gitlab, NPM, and any other sites that render the readme etc... 

Note that we can expand the definition of canonical sources to include Gitlab etc... as we find projects that want to list their primary URL as being from those domains. For example:

```ruby
 def restrict_to_sponsor_campaigns?
  %w[github.com gitlab.com bitbucket.org].include? host
end
```